setup.py

```diff
@@ -21,8 +11,25 @@
     license='GPLv3+',
     packages=find_packages(exclude=['tests']),
     include_package_data=True,
-    install_requires=requirements,
-    dependency_links=dependency_links,
+    install_requires=[
+        'flask',
+        'future',
+        'wheel',
+        'estuary @ https://github.com/release-engineering/estuary-api/tarball/master#egg=estuary',
+        'fedmsg',
+        'fedmsg[commands]',
+        'fedmsg[consumers]',
+        'requests',
+        'requests_kerberos',
+        'koji',
+        'moksha.hub',
+        'PyOpenSSL',
+        'stomper',
+        # Pin the version until this is merged:
+        # https://github.com/neo4j-contrib/neomodel/pull/553
+        'neomodel==3.3.2',
+        'typing-extensions'
```

Comment:
What is this for?

Reply:
I think `typing-extensions` was required by tests, so I added it here, but then also added it to test-requirements too, so I can delete it.

Reply:
And `neomodel` required by estuary, but I saw you have pinned version in estuary-api, so I thought I need to add it here too.